### ISSUE TITLE
Fold using heading cells

### ISSUE THREAD

**Issue by jswhite97** [tkf/emacs-ipython-notebook#113]
It would be interesting to be able to fold/unfold a long notebook using the heading cells (e.g., like code folding, but displaying only the heading cells when folded).  I know this is probably not trivial, but I wanted to throw it out there as an idea.

**Comment by tkf** [comment]
I agree that would be useful but for time being I have no plan to implement by myself.  If somebody wants to implement it, patch is very welcome.

Until then, you can use "imenu viewer" like e2wm (https://github.com/tkf/emacs-window-manager#imenu) to see overview of the notebook.   I know it is not equivalent as folding but it shares some function.
